src/envoy/http/path_matcher/filter_config.cc

```diff
@@ -35,18 +35,14 @@ FilterConfig::FilterConfig(
       throw Envoy::ProtoValidationException(
           "Duplicated pattern or invalid pattern", rule.pattern());
     }
-    if (rule.extract_path_parameters()) {
-      path_params_operations_.insert(rule.operation());
+
+    // Only stores the operations that need path param extraction.
+    if (rule.has_path_parameter_extraction()) {
```

Comment:
We don't need another map here.  We can pass in the pointer to &rule in line 34.  It will have both operation and extraction.

Reply:
I see, thanks. It's cleaner now.